### DISCUSSION TITLE
blog/2023-04-24-causal-inference-with-logistic-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#14] [Comments]
# blog/2023-04-24-causal-inference-with-logistic-regression/

Part 3 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-04-24-causal-inference-with-logistic-regression/

**Comment by marcdotson** [comment]
Hi, Solomon. Awesome post. One small correction: the inverse of the `logit()` link function *is* the logistic function (i.e., `plogis()`). In other words, `logit()` is the link function, `logistic()` is the inverse link function, hence it being called both the logit model and logistic regression.

**Reply by ASKurz** [reply]
Okay @marcdotson, I believe I've made the edits necessary to account for your correction. Please let me know if you find other bits needing an update. Cheers!

**Comment by ASKurz** [comment]
For a nice new paper on how to adjust for covariates used during the stratified randomization phase (discussed in endnote #5), see Sullivan et al (2024; [https://doi.org/10.1002/sim.10060](https://doi.org/10.1002/sim.10060)), *Categorisation of continuous covariates for stratified randomisation: How should we adjust?*
